package.json

```diff
@@ -6,14 +6,18 @@
   "repository": "git://github.com/airtable/airtable.js.git",
   "private": false,
   "scripts": {
-    "coverage": "jest --env node --coverage",
-    "pretest": "npm run lint",
-    "lint": "eslint .",
+    "coverage": "jest --env node --coverage --no-cache",
+    "pretest": "npm run lint && npm run compile; true",
+    "lint": "eslint '*/**/*.{js,ts,tsx}'",
     "format": "prettier --write '**/*.js'",
-    "test": "npm run coverage && npm run test-unit",
-    "test-unit": "jest --env node"
+    "test": "npm run coverage",
+    "test-unit": "jest --env node",
+    "compile": "rm -rf build/dist/* && tsc",
+    "browserify": "npm run compile; grunt browserify"
   },
   "dependencies": {
+    "@types/node": "^14.0.14",
+    "@typescript-eslint/parser": "^3.4.0",
```

Comment:
We do. Otherwise I get an error:

```
Error: Failed to load parser '@typescript-eslint/parser' declared in '.eslintrc.json » plugin:@typescript-eslint/recommended » ./configs/base$
: Cannot find module '@typescript-eslint/parser'                   
Require stack:
- /home/rebecca/airtable/airtable.js/node_modules/@typescript-eslint/eslint-plugin/dist/configs/base.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.resolve (internal/modules/cjs/helpers.js:83:19)
    at Object.resolve (/home/rebecca/airtable/airtable.js/node_modules/eslint/lib/shared/relative-module-resolver.js:44:50)
    at ConfigArrayFactory._loadParser (/home/rebecca/airtable/airtable.js/node_modules/eslint/lib/cli-engine/config-array-factory.js:870:45)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/home/rebecca/airtable/airtable.js/node_modules/eslint/lib/cli-engine/config-array-f
actory.js:665:32)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/home/rebecca/airtable/airtable.js/node_modules/eslint/lib/cli-engine/config-array-facto
ry.js:596:20)
    at _normalizeObjectConfigData.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/home/rebecca/airtable/airtable.js/node_modules/eslint/lib/cli-engine/config-array-f
actory.js:660:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)

```